developer_guide/installation/index.rst

```diff
@@ -0,0 +1,18 @@
+Installation
+============
+
+Once you have checked that your system requirements fulfill the needs of the PIM, you are ready to install it. If you want to easily setup the system requirements, see the *Setup System Requirements on* pages.
```

Comment:
Once the requirements are fullfiled, you are ready to install Akeneo PIM. Otherwise, check *Setup System Requirements*